solidjs-typescript/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="module" src="/src/index.tsx"></script>
+    <script src="/datafeeds/udf/dist/bundle.js"></script>
+  </body>
+</html>
```

Comment:
Adding a newline at end of file.

```suggestion
</html>

```